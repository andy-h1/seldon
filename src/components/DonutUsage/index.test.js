import React from 'react';
import { render } from '@testing-library/react';
import { DonutUsage } from '.';

const mockData = {
  name: 'Youtube Views',
  usage: 750,
  total: 1000,
  color: 'red',
};

describe('Usage Donut component', () => {
  it('should render', () => {
    const { container } = render(<DonutUsage data={mockData} />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
