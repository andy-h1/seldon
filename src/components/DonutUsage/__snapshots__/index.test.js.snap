// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Usage Donut component should render 1`] = `
<div
  class="sc-hKgILt hltTKj"
>
  <svg
    height="100%"
    viewBox="0 0 100 100"
    width="100%"
  >
    <circle
      class="sc-gsTCUz cRaPmX"
      cx="50"
      cy="50"
      r="45"
      transform="rotate (-90 50 50)"
    />
    <circle
      class="sc-jSgupP gdSHyo"
      color="red"
      cx="50"
      cy="50"
      r="45"
      transform="rotate (-90 50 50)"
    />
    <text
      class="sc-eCssSg kPwlTr"
      x="50%"
      y="50%"
    >
      750
    </text>
    <defs>
      <lineargradient
        id="donutFill"
      >
        <stop
          offset="0%"
          stop-color="#83f9b0"
        />
        <stop
          offset="100%"
          stop-color="#cbf07d"
        />
      </lineargradient>
    </defs>
  </svg>
  <p
    class="MuiTypography-root MuiTypography-body1"
  >
    Youtube Views
     used
  </p>
  <p
    class="MuiTypography-root MuiTypography-body1"
  >
    250
    /
    1000
     left
  </p>
</div>
`;
